src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -698,52 +682,10 @@ classes:
     - object_preprocessing
     - semantic_similarity_score
     - semantic_similarity_measure
-    - see_also
-    - issue_tracker_item
-    - other
-    - comment
-    class_uri: owl:Axiom
-  literal mapping:
-    description: Represents an individual mapping between a literal and an entity. 
-      Note that this schema has been created on 01.08.2023 and is subject to change.
-    see_also:
-    - https://mapping-commons.github.io/sssom/sssom-profiles/
-    slots:
-    - literal
-    - literal_datatype
-    - predicate_id
-    - predicate_label
-    - predicate_modifier
-    - object_id
-    - object_label
-    - object_category
-    - mapping_justification
-    - author_id
-    - author_label
-    - reviewer_id
-    - reviewer_label
-    - creator_id
-    - creator_label
-    - license
-    - literal_source
-    - literal_source_version
-    - object_type
-    - object_source
-    - object_source_version
-    - mapping_provider
-    - mapping_source
-    - mapping_cardinality
-    - mapping_tool
-    - mapping_tool_version
-    - mapping_date
-    - confidence
-    - object_match_field
-    - match_string
-    - literal_preprocessing
-    - object_preprocessing
     - similarity_score
     - similarity_measure
```

Comment:
The diff is conveniently obfuscating the fact that these two slots are _added_ to the mappings class. They should have been there from the start anyways, but just being transparent.